server/server-kv.c

```diff
@@ -20,11 +21,56 @@
         http://beej.us/guide/bgnet/
 ****************************************/
 
+#define DEBUG
+
+int flag_test;
+
 struct pool_list *list_head = NULL, *list_tail = NULL;
 
 pthread_mutex_t mutex = PTHREAD_MUTEX_INITIALIZER;
 pthread_cond_t cond = PTHREAD_COND_INITIALIZER;
 pthread_t threads[MAX_CONCURRENCY];
+struct gwkv_server* server;
+
+void
+process_operation(struct gwkv_server *server, struct operation *op, char *message)
```

Comment:
I'm pretty sure @nks5295 has already written code to do this in #49. What's the purpose of this?